pac4j-oauth/src/main/java/org/pac4j/oauth/profile/weibo/WeiboProfileDefinition.java

```diff
@@ -0,0 +1,240 @@
+package org.pac4j.oauth.profile.weibo;
+
+import java.util.Arrays;
+
+import org.pac4j.core.exception.HttpAction;
+import org.pac4j.core.profile.ProfileHelper;
+import org.pac4j.core.profile.converter.Converters;
+import org.pac4j.oauth.config.OAuth20Configuration;
+import org.pac4j.oauth.profile.JsonHelper;
+import org.pac4j.oauth.profile.definition.OAuth20ProfileDefinition;
+import org.pac4j.scribe.model.WeiboToken;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.github.scribejava.core.exceptions.OAuthException;
+import com.github.scribejava.core.model.OAuth2AccessToken;
+
+import static org.pac4j.core.profile.AttributeLocation.PROFILE_ATTRIBUTE;
+
+/**
+ * This class is the  Sina Weibo profile definition (using OAuth 2.0 protocol).
+ * <p>
+ * <p>More info at: <a href="http://open.weibo.com/wiki/2/users/show">users/show</a></p>
+ *
+ * @author Zhang Zhenli
+ * @since 3.1.0
+ */
+public class WeiboProfileDefinition extends OAuth20ProfileDefinition<WeiboProfile, OAuth20Configuration> {
+
+    /**
+     * int64    User UID
+     */
+    public static final String id = "id";
```

Comment:
I think all constants could be uppercase (to follow convention).

Reply:
This is my lazy handling,。 
too many variables, the official documentation I copied directly